api/middleware/middleware.go

```diff
@@ -37,6 +37,9 @@ func getOrg(req *http.Request, multiTenant bool) (int, error) {
 	}
 	orgStr := req.Header.Get("x-org-id")
 	if orgStr == "" {
+		orgStr = req.Header.Get("x-grafana-org-id")
+	}
+	if orgStr == "" {
```

Comment:
@woodsaj thoughts on this? I did it because grafana uses that header now, and to make the dev stack work where grafana talks to MT directly